modules/150-user-authn/templates/dex/config.yaml

```
@@ -19,6 +19,23 @@
     extra:
       # TODO(nabokihms): make it configurable
       lang: en
+  {{- with $context.Values.userAuthn.passwordPolicy }}
+    {{ if . }}
+    passwordPolicy:
```

```suggestion
  passwordPolicy:
```

There is an error with indents, config renders with extra two spaces and the feature doesn't work.